tox.ini

```diff
@@ -97,6 +97,7 @@ commands =
     lint: pylint --rcfile=tests/pyproject.toml tests
     lint: pydocstyle lms tests bin
```

Comment:
I have made no attempt to replace isort/black with ruff. That's seems pretty straightforward.